### PR TITLE
fix-zh-home

### LINKED ISSUE #14830: No content on /zh/docs/home/
**This is a Feature Request**

**What would you like to be added**
https://kubernetes.io/zh/docs/home/ (docs home page for 中文 / Chinese localization) is essentially blank.

**Why is this needed**
Visitors to this page would expect substantially more information.

**Comments**
This page has recently been showing a low score for visitor satisfaction (via feedback buttons on the page).
/language zh

### PR DESCRIPTION
**Pull request by zhangqx2010**
fix #14830

### CODE REVIEW

File: content/zh/docs/home/_index.md [hunk 1]
```diff
@@ -1,42 +1,94 @@
...
   main:
     title: "文档"
     weight: 20
     post: >
       <p>通过演练，示例和参考文档了解如何使用 Kubernetes。你甚至可以<a href="/editdocs/" data-auto-burger-exclude>帮助贡献文档</a>！</p>
+overview: >
+  <!-- Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation (<a href="https://www.cncf.io/about">CNCF</a>). -->
+  Kubernetes 是一个开源容器编排引擎，用于容器化应用的自动化部署、扩展和管理。该项目托管在 <a href="https://www.cncf.io/about">CNCF</a>。
+<!-- cards:
+- name: concepts
```
Comment (chenrui333) [inline content/zh/docs/home/_index.md]: Do you have any reference for such changes? Like the layout of the file?

Comment (zhangqx2010) [reply content/zh/docs/home/_index.md]: Seems work fine by netlify preview.
It's strange that the doc in release-1.14 the md looks fine, while in master the content is missing.
```
https://raw.githubusercontent.com/kubernetes/website/release-1.14/content/zh/docs/home/_index.md
vs
https://raw.githubusercontent.com/kubernetes/website/master/content/zh/docs/home/_index.md
```
Maybe some pr changed this by mistake?

Comment (zhangqx2010) [reply content/zh/docs/home/_index.md]: Also the file in master looks like in old style format.

**Comment by zhangqx2010** [comment]
@chenrui333 One thing comes to my mind that we tranlate based on release-1.14 branch now. So many change won't come to the live website. Maybe we should discuss with @markthink and change the process a little?
I am not sure the exact process of branch management. When will the master merge to the release branch and will it produce conflict with our translated file?

**Comment by chenrui333** [comment]
@zhangqx2010 @markthink 

We can just periodically refresh the release-1.14 to master branch regarding `content/zh` folder. Like every month, which should be totally doable.

**Pull request merged**